src/app/store.js

```diff
@@ -0,0 +1,12 @@
+import { configureStore } from '@reduxjs/toolkit';
+import cryptoReducer from '../feature/cryptoSlice';
+import searchReducer from '../feature/searchSlice';
+
+const store = configureStore({
+  reducer: {
+    crypto: cryptoReducer,
+    search: searchReducer,
+  },
+});
+
+export default store;
```

Comment:
- [ ] Please as I requested in the previous review it would be better to put all your redux code in a folder called `Redux`  in order to follow the best practices and also will be easy for other developers to read your code